invokeai/frontend/web/src/features/system/components/SiteHeader.tsx

```diff
@@ -34,87 +33,58 @@ const SiteHeader = () => {
 
-      <HotkeysModal>
-        <IAIIconButton
-          aria-label={t('common.hotkeysLabel')}
-          tooltip={t('common.hotkeysLabel')}
-          size="sm"
-          variant="link"
-          data-variant="link"
-          fontSize={20}
-          icon={<FaKeyboard />}
-        />
-      </HotkeysModal>
-
-      {isLocalizationEnabled && <LanguagePicker />}
-
-      {isBugLinkEnabled && (
-        <Link
-          isExternal
-          href="http://github.com/invoke-ai/InvokeAI/issues"
-          marginBottom="-0.25rem"
-        >
-          <IAIIconButton
-            aria-label={t('common.reportBugLabel')}
-            tooltip={t('common.reportBugLabel')}
-            variant="link"
-            data-variant="link"
-            fontSize={20}
-            size="sm"
-            icon={<FaBug />}
-          />
-        </Link>
-      )}
-
-      {isGithubLinkEnabled && (
-        <Link
-          isExternal
-          href="http://github.com/invoke-ai/InvokeAI"
-          marginBottom="-0.25rem"
-        >
-          <IAIIconButton
-            aria-label={t('common.githubLabel')}
-            tooltip={t('common.githubLabel')}
-            variant="link"
-            data-variant="link"
-            fontSize={20}
-            size="sm"
-            icon={<FaGithub />}
-          />
-        </Link>
-      )}
-
-      {isDiscordLinkEnabled && (
-        <Link
-          isExternal
-          href="https://discord.gg/ZmtBAhwWhy"
-          marginBottom="-0.25rem"
-        >
-          <IAIIconButton
-            aria-label={t('common.discordLabel')}
-            tooltip={t('common.discordLabel')}
-            variant="link"
-            data-variant="link"
-            fontSize={20}
-            size="sm"
-            icon={<FaDiscord />}
-          />
-        </Link>
-      )}
-
-      <ColorModeButton />
-
-      <SettingsModal>
-        <IAIIconButton
-          aria-label={t('common.settingsLabel')}
-          tooltip={t('common.settingsLabel')}
+      <Menu>
+        <MenuButton
+          as={IAIIconButton}
           variant="link"
-          data-variant="link"
-          fontSize={22}
-          size="sm"
-          icon={<MdSettings />}
+          aria-label={t('accessibility.menu')}
+          tooltip={t('accessibility.menu')}
+          icon={<FaBars />}
         />
-      </SettingsModal>
+        <MenuList>
+          <MenuGroup title={t('common.communityLabel')}>
+            {isGithubLinkEnabled && (
+              <MenuItem
+                as="a"
+                href={githubLink}
+                icon={<FaGithub fontSize={20} />}
+              >
+                {t('common.githubLabel')}
+              </MenuItem>
+            )}
+            {isBugLinkEnabled && (
+              <MenuItem
+                as="a"
+                href={`${githubLink}/issues}`}
+                icon={<FaBug fontSize={20} />}
+              >
+                {t('common.reportBugLabel')}
+              </MenuItem>
```

Comment:
`MenuItems` that are links need to be done like this to provide an accessible UI:
```ts
<Link href={imageDTO.image_url} target="_blank">
  <MenuItem icon={<FaExternalLinkAlt />}>
    {t('common.openInNewTab')}
  </MenuItem>
</Link>
```

Reply:
also IMO all the links should open in a new tab, right now they open in the same tab

Reply:
Fixed.